vms/platformvm/state/state.go

```diff
@@ -48,9 +52,15 @@ const (
 	pointerOverhead = wrappers.LongLen
 
 	validatorDiffsCacheSize = 2048
+	blockIDCacheSize        = 2048
 	rewardUTXOsCacheSize    = 2048
 	chainCacheSize          = 2048
 	chainDBCacheSize        = 2048
+
+	pruneCommitLimit           = 1024
+	pruneCommitSleepMultiplier = 5
+	pruneCommitSleepCap        = 10 * time.Second
+	pruneUpdateFrequency       = 30 * time.Second
```

Comment:
nit: replace `pruneUpdateFrequency` with  `pruneUpdatePeriod`

Reply:
`pruneUpdateFrequency` is in-line with the `avm` state pruning code

Reply:
still it's a period :-). But ok, we can clean them up altogether later on